webapp/src/reducers/index.js

```diff
@@ -95,9 +86,9 @@ function reviewsDetails(state = [], action) {
     }
 }
 
-function yourPrs(state = [], action) {
+function sidebarContent(state = [], action) {
```

Comment:
The default value for `sidebarContent` should be this:

```suggestion
const defaultSidebarContent = {
    reviews: [],
    prs: [],
    assignments: [],
    unreads: [],
}

function sidebarContent(state = defaultSidebarContent, action) {
```